doc/sphinx/user/benchmarks/benchmarks

```diff
@@ -0,0 +1 @@
+/home/chlmills/aspect/benchmarks
```

Comment:
This seems incorrect.

Reply:
Thanks for the catch, we were stumped. I set up the symlink wrong, we're trying it again